services/ui-src/src/utils/forms/forms.ts

```diff
@@ -42,6 +42,7 @@ export const formFieldFactory = (
       key: field.id,
       name: field.id,
       hydrate: field.props?.hydrate,
+      autoComplete: "one-time-code",
```

Comment:
Nit: Can we add a note, like:
```suggestion
      autoComplete: "one-time-code", // stops browsers from forcing autofill
```

Reply:
We sure can